test/invariants/handlers/RouterHandler.sol

```diff
@@ -95,12 +112,68 @@ contract RouterHandler is Test, TypedDataSignature {
     function executeWithSignerFee(
         uint256 actorSeed
     ) external useActor(actorSeed) recordAgent countCall('executeWithSignerFee') {
-        vm.startPrank(currentActor);
         uint256 deadline = block.timestamp;
         DataType.LogicBatch memory logicBatch = DataType.LogicBatch(logicsEmpty, feesEmpty, referralsEmpty, deadline);
         bytes memory sigature = getTypedDataSignature(logicBatch, router.domainSeparator(), signerPrivateKey);
+        vm.prank(currentActor);
         router.executeWithSignerFee(permit2DatasEmpty, logicBatch, signer, sigature, tokensReturnEmpty);
-        vm.stopPrank();
+    }
+
+    function executeBySig() external useAddress(user) recordAgent countCall('executeBySig') {
+        // Ensure correct EIP-712 encodeData
```

Comment:
Could you help remove this comment, as well as those in `Router.t.sol`? The correct encoding is provided in the `*Typehash.t.sol` files instead of having `*Empty` fields.